ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -2,26 +2,26 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 }}
-{{#unless @isActivated}}
-  {{#if (or @licenseHasSecretsSync @isHvdManaged)}}
+{{#unless this.flags.secretsSyncIsActivated}}
+  {{#if this.version.hasSecretsSync}}
     {{#unless this.hideOptIn}}
       {{! Allows users to dismiss activation banner if they have permissions to activate. }}
       <Hds::Alert
         @type="inline"
         @color="warning"
-        @onDismiss={{if this.flags.canActivateSecretsSync (fn (mut this.hideOptIn) true) undefined}}
+        @onDismiss={{if @canActivateSecretsSync (fn (mut this.hideOptIn) true) undefined}}
```

Comment:
I did move permissions to the route model since this was the "cleanest" way to use `lazyCapabilities` without too much of a rabbit whole refactor